proto/src/determined/master/v1/master.proto

```diff
@@ -97,7 +98,7 @@ message LogConfig {
     json_schema: { required: [] }
   };
   // The log level for Master Config.
-  string level = 1;
+  determined.log.v1.LogLevel level = 1;
```

Comment:
fyi, this enum usually maps to python log levels not logrus so it's not exactly perfect. but i think it is fine,